src/GraphQL/Types/Fields/FieldType.cs

```diff
@@ -93,6 +94,11 @@ public Type? Type
     /// </summary>
     public ISourceStreamResolver? StreamResolver { get; set; }
 
+    /// <summary>
+    /// Gets or sets field middleware for the field. Only applicable to fields of output graph types.
+    /// </summary>
+    public IFieldMiddleware? Middleware { get; set; }
```

Comment:
There is a single new field defined on `FieldType`.  It reuses the same `IFieldMiddleware` interface that already exists within the project.  The extension method `ApplyMiddleware` will chain middleware together if called multiple times.  But for the majority of use cases, this will be null or set to `ScopedFieldMiddleware.Instance`.